### PR TITLE
Feature#3 login formik

### LINKED ISSUE #3: Login Form with Formik
# REQUIREMENTS
The login should be made with formik.

# DESIGN
LogIn (1)

# Acceptance Criteria
The design should be similar a the proposal.

### PR DESCRIPTION
**Pull request by HRuvalcabaP**
# Description
-We created login form with formik.
-We created the respective validations.
-We added format with prittier 

  
  ## Screenshots 

https://user-images.githubusercontent.com/50897282/113610657-09ecf000-9613-11eb-9299-143d72fd424a.mp4

image

### CODE REVIEW

**Review by GracielaJazmin — DISMISSED** [review]
Good job!

**Review by vmagana3 — DISMISSED** [review]
Good work using formik to validate and define initial values on form!!

File: AlertCovidApp/src/components/atoms/LogIn/InputLogIn.js [hunk 1]
```diff
@@ -16,7 +24,16 @@ const InputLogIn = ({PlaceHolderText, iconName, text}) => {
...
-        secureTextEntry={iconName === 'lock' && true}></TextInput>
+        secureTextEntry={iconName === 'lock' && true}
+        onChangeText={(text) => {
+          if (PlaceHolderText === 'Email') {
+            setUserEmail('email', text);
+          } else {
+            setUserPass('password', text);
+          }
+        }}
+        //onChangeText ={text => setFieldValue(label, text)}
```
Comment (LeyaimMagma) [inline AlertCovidApp/src/components/atoms/LogIn/InputLogIn.js]: delete this comment

File: AlertCovidApp/src/components/atoms/Symptoms/CheckboxSymptoms.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import {CheckBox} from 'react-native-elements';
+
+
```
Comment (LeyaimMagma) [inline AlertCovidApp/src/components/atoms/Symptoms/CheckboxSymptoms.js]: Remove one space

File: AlertCovidApp/src/screens/LogIn.js [hunk 3]
```diff
@@ -7,8 +7,46 @@ import CreateAccountText from '../components/atoms/LogIn/CreateAccountText';
...
+      password: '',
+    },
+    onSubmit: (values) => {
+      //Aqui se hace la petición al endpoint de la API//
+      console.log(values.email);
+      console.log(values.password);
+      //fetch('http://example.com/movies.json')
+      //.then(response => response.json())
+      //.then(data => console.log(data));
+    },
```
Comment (LeyaimMagma) [inline AlertCovidApp/src/screens/LogIn.js]: ??, Remove console.logs

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by LeyaimMagma — APPROVED** [review]

**Review by GracielaJazmin — APPROVED** [review]

**Review by vmagana3 — APPROVED** [review]
Good work using formik to validate!!

**Pull request merged**